e2e/eth-3-test.toml

```diff
@@ -4,7 +4,7 @@
 
 # Perform a transaction so can complete the first block
 [[TestCases]]
-  RunCmd = "{{ $.LoomPath }} deploy -b SimpleStore.bin -n SimpleStore -k {{index $.AccountPrivKeyPathList 0}}"
+  RunCmd = "{{ $.LoomPath }} deploy -b SimpleStore.bin -n SimpleStore  -v 1844674407370955161500000 -k {{index $.AccountPrivKeyPathList 0}}"
```

Comment:
This test command fails with insufficient fund error. Should we fix it?

Reply:
No reason it needs to work.

Reply:
ok